api/applications/applications.proto

```diff
@@ -0,0 +1,28 @@
+
+syntax = "proto3";
+
+package wego_server.v1;
+
+option go_package = "github.com/weaveworks/weave-gitops/pkg/protos/applications";
+
+import "google/api/annotations.proto";
+
+service Applications {
+    rpc ListApplications(ListApplicationsReq) returns (ListApplicationsRes) {
+        option (google.api.http) = {
+            get : "/v1/applications"
```

Comment:
Following the [Google naming conventions](https://cloud.google.com/apis/design/naming_convention#request_and_response_messages) how about we change these to `ListApplicationsRequest` and `ListApplicationsResponse` ?

Reply:
Also, let's put comments above the methods?

Something like...

```protobuf
service Applications {
    /**
     * ListApplications returns the list of WeGo applications that the authenticated user has access to.
     */
    rpc ListApplications(ListApplicationsRequest) returns (ListApplicationsResponse) {
        option (google.api.http) = {
            get : "/v1/applications"
          };
    }
}
```

These are output in the Swagger, and in the generated documentation.

Reply:
Added comments and changed the req/res type names